### PR TITLE
CAKE-5209 Design updates and fixes for search results interrupt

### PR DESCRIPTION
**Pull request by mklucsarits**
## Links
* http://wikia-inc.atlassian.net/browse/CAKE-5209

## Description
 * Fixed styling issues to match mocked up design
 * Removed extraneous div elements around components

## Reviewers
@Wikia/cake

### CODE REVIEW

**Review by vforge — APPROVED** [review]

File: app/components/post-search-results-affiliate-item.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import Component from '@ember/component';
 import { track, trackActions } from '../utils/track';
 
 export default Component.extend({
+  classNames: ['post-search-results-affiliate-item'],
```
Comment (JoshuaRogan) [inline app/components/post-search-results-affiliate-item.js:6]: Interesting. Does that mean there is always a div for each component?

Comment (vforge) [reply app/components/post-search-results-affiliate-item.js:6]: Yeah, ember takes care of it

**Comment by vforge** [comment]
```

mobile-wiki/templates/components/post-search-results.hbs

  32:6  error  Incorrect indentation for `{{#each}}` beginning at L32:C6. Expected `{{#each}}` to be at an indentation of 8 but was found at 6.  block-indentation
```

File: app/components/post-search-results-affiliate-item.js (hunk 1 above)
Comment (JoshuaRogan) [reply app/components/post-search-results-affiliate-item.js:6]: Hmmm. I'm not sure if I like that or find it to be wasteful if you have the template.

Comment (vforge) [reply app/components/post-search-results-affiliate-item.js:6]: Regardless it will always* be there.

**Pull request merged**